src/client/offline.html

```diff
@@ -0,0 +1,308 @@
+<!doctype html>
```

Comment:
Required: Static files should live in the `public/`

Reply:
As mentioned above, making some tweaks to get things setup correctly is good with me! Even with a little bit of scope creep.

The comment was about putting the source file in `public/` folders, and then copying it over to `/build/client` using webpack. Which I believe is now happening here: https://github.com/politics-rewired/Spoke/pull/1411/files#diff-ec2cabc8139e814d33682814c1ba15a1056d946ab70f740c9d42ada07af42050R41
This way all static files that we need to copy (in this case, a copy webpack plugin might be better than a html webpack plugin) are copied from `public/` into `PUBLIC_DIR`